application/src/main/java/org/opentripplanner/framework/graphql/GraphQLUtils.java

```diff
@@ -42,18 +49,10 @@ public static Locale getLocale(DataFetchingEnvironment environment, Locale local
 
   public static Locale getLocaleFromEnvironment(DataFetchingEnvironment environment) {
     // This can come from the accept-language header
-    var userLocale = environment.getLocale();
-    var defaultLocale = getDefaultLocale(environment);
-
-    if (userLocale == null) {
-      return defaultLocale.orElse(Locale.forLanguageTag("*"));
-    }
-
-    if (defaultLocale.isPresent() && acceptAnyLocale(userLocale)) {
-      return defaultLocale.get();
-    }
-
-    return userLocale;
+    var envLocale = environment.getLocale();
```

Comment:
Just a question about names:

- localContextLocale is the one passed in the GraphQL query body
- env Locale is passed the one from the HTTP header `Accept-Language`

Correct?

If so, can you write a few lines in this class? It would help me understand what is going on.

Reply:
There is some comments in this method that explain this. But I'll clarify the comments a bit and add javadoc to methods.